### ISSUE TITLE
More informative blocked post indicator

### ISSUE THREAD

**Issue by scottdotweb** [bluesky-social/social-app#7078]
### Describe the Feature

Currently a blocked post, when quoted, will appear as "ℹ️ Blocked" and nothing else. It would be more useful if the indicator specifies the blocking direction:
- Quoter blocked by post's author (not identified)
- You blocked the author (identified)
- The author (not identified) blocked you 
- A moderation list you subscribe to (identified) blocked the author (identified)

When the block comes from the post's author the "nuclear block" privacy level is maintained, but when it comes from you you're given an actionable record.

**Comment by hexdotooo** [comment]
This is a duplicate of #2127. Author has gone so a team member should close this.

**Comment by meg-huggingface** [comment]
Was about to reference this from finding it in search; please close.
